### ISSUE TITLE
[BUG] Javascript error when manipulating a boltauth user

### ISSUE THREAD

**Issue by panda-madness** [BoltAuth/Auth#9]
A Javascript error occurs when trying to enable/disable/delete a user or add/remove a role.
The error is `auth is not defined`

Console output below:
```
Deleting user(s): 739ebb13-8d53-4d70-89c9-4cb0f1e9b914  auth-admin.js:93:13
ReferenceError: auth is not defined[Learn More]  bolt.js:1:1
```

Details
-------

 - Relevant Bolt Auth Version: [ master ]
 - Relevant Bolt Version: [ 3.3.1 ]
 - PHP version: [ 7.1.0 ]
 - For UX/UI issues: [ Firefox DevEdition 55 ]

Reproduction
------------
Create a new boltauth user, then try to enable/disable/delete him or add/remove a role.

**Comment by rossriley** [comment]
Give it a check but I think #8 solved this one.

**Comment by SvanteRichter** [comment]
Seems to be fixed in #8, thanks rossriley!
